library_schemas/score/prerelease/HED_score_1.1.1.mediawiki

```diff
@@ -12,953 +12,587 @@ For more information see https://hed-schema-library.readthedocs.io/en/latest/ind
-            *** ECG-bradycardia <nowiki> {suggestedTag=Frequency} </nowiki>
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-            *** ECG-extrasystole
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-            *** ECG-ventricular-premature-depolarization <nowiki> {suggestedTag=Frequency} </nowiki>
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-            *** ECG-tachycardia <nowiki> {suggestedTag=Frequency} </nowiki>
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-            *** Other-ECG-feature {requireChild}
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-    * EMG-channel-finding <nowiki> {suggestedTag=Finding-significance-to-recording} [EMG channel findings involve distinctive muscular activity patterns and characteristics observed in the Electromyography recording.]</nowiki>
-        ** EMG-muscle-side
-            *** EMG-left-muscle
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-            *** EMG-right-muscle
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-            *** EMG-bilateral-muscle
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
+    * ECG-channel-finding <nowiki> {suggestedTag=Finding-significance-to-recording} [Findings in Electrocardiogram recordings. (Source: Beniczky ea 2017, Section 13, Table 16.)]</nowiki>
+        ** ECG-QT-period <nowiki> {requireChild}[Value and free text. (Source: Beniczky ea 2013, Table 9; Beniczky ea 2017, Table 16.)] </nowiki>
+            *** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
+        ** ECG-normal-rhythm <nowiki> {suggestedTag=Frequency} [Normal rhythm. (Source: Beniczky ea 2017, Table 16.)]</nowiki>
+            *** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
+        ** ECG-arrhythmia <nowiki> [Source: Beniczky ea 2017, Table 16.] </nowiki>
+            *** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
+        ** ECG-asystolia <nowiki>[Duration (range in seconds). (Source: Beniczky ea 2013, Table 9.)]</nowiki>
+            *** <nowiki># {takesValue, valueClass=numericClass} </nowiki>
+        ** ECG-bradycardia <nowiki> {suggestedTag=Frequency}[Numerical value for frequency in beats/minute. (Source: Beniczky ea 2017, Table 16; Beniczky ea 2013, Table 9.)] </nowiki>
+            *** <nowiki># {takesValue, valueClass=numericClass} </nowiki>
+        ** ECG-extrasystole <nowiki> [Source: Beniczky ea 2017, Table 16.] </nowiki>
+            *** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
+        ** ECG-ventricular-premature-depolarization <nowiki> [Source: Beniczky ea 2017, Table 16.] </nowiki>
```

Comment:
I didn't understand the what this should be from googling.  Could we add a description here?

Reply:
Added: "A premature ventricular contraction (PVC) is a common event where the heartbeat is initiated by Purkinje fibers in the ventricles rather than by the sinoatrial node. (Source: Wikipedia; Beniczky ea 2017, Table 16.)"